src/Evolution/Executables/GrMhd/ValenciaDivClean/EvolveValenciaDivClean.hpp

```diff
@@ -99,13 +100,8 @@ struct EvolutionMetavars {
       grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::NewmanHamlin,
       grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::PalenzuelaEtAl>;
 
-  // hack this has to be synchronized with the Observe action :(
-  using Redum = Parallel::ReductionDatum<double, funcl::Plus<>,
-                                         funcl::Sqrt<funcl::Divides<>>,
-                                         std::index_sequence<1>>;
-  using reduction_data_tags = tmpl::list<observers::Tags::ReductionData<
-      Parallel::ReductionDatum<double, funcl::AssertEqual<>>,
-      Parallel::ReductionDatum<size_t, funcl::Plus<>>, Redum, Redum, Redum>>;
+  using reduction_data_tags = observers::get_reduction_data_tags_from_observers<
```

Comment:
This isn't getting from the observers, it's getting it from the action that sends data to the observers. I think it would be good to keep those two concepts distinct. I think the observers actually use this type alias to get what will be observed.